round4/team-07.md

```diff
@@ -0,0 +1,156 @@
+# Chrome Extension Idea: Currency Converter
+
+## Authors
+
+Akanksha Tripathy, Ruth Tilahun, Tianyi Wu, Yola Yan
+
+## Problem Statement - TW
+
+[Describe the problem that your Chrome Extension will solve. What are the pain points that users experience? How does your extension address these pain points?]
```

Comment:
@akankshatripathy Please remove the instructions contained in brackets from the entire document.